src/main/scala/com/cognite/sdk/scala/v1/dataModelProperties.scala

```diff
@@ -14,3 +14,39 @@ final case class Float64Property(value: Double) extends PropertyTypePrimitive
 final case class StringProperty(value: String) extends PropertyTypePrimitive
 
 final case class ArrayProperty[+A <: PropertyTypePrimitive](values: Vector[A]) extends PropertyType
+
+// These types below are treated as string for now
+final case class DirectRelationProperty(value: String) extends PropertyType
+final case class TimeStampProperty(value: String) extends PropertyType
```

Comment:
maybe we should parse this into Instant or sth? as we do for `lastUpdatedTime` etc, just an idea?

Reply:
because spark supports timestamp type, would be cool to recognize it.

Reply:
Updated, `TimeStampProperty` uses `java.ZonedDateTime` (because imestamps in DMS are timezone-aware) and `DateProperty` uses` java.LocalDate`